File: app/src/main/java/com/lineclient/home/homelineclient/tools/ViewTool.java
package com.lineclient.home.homelineclient.tools;

import android.content.Context;
import android.util.DisplayMetrics;

public class ViewTool {

    public static int getScreenWidth(Context context){
        DisplayMetrics dm2 = context.getResources().getDisplayMetrics();
        return dm2.widthPixels;
    }

    public static int getScreenHeight(Context context){
        DisplayMetrics dm2 = context.getResources().getDisplayMetrics();
        return dm2.heightPixels;
    }

}

File: app/src/main/java/com/lineclient/home/homelineclient/tools/ViewTool.kt
package com.lineclient.home.homelineclient.tools

import android.content.Context

object ViewTool {
    @JvmStatic fun getScreenWidth(context: Context): Int {
        val dm2 = context.resources.displayMetrics
        return dm2.widthPixels
    }

    fun getScreenHeight(context: Context): Int {
        val dm2 = context.resources.displayMetrics
        return dm2.heightPixels
    }
}